assets/js/nav.js

```diff
@@ -0,0 +1,17 @@
+const navChevron = document.querySelectorAll(".chevron-down");
+const chevronUp = "/theme-assets/chevron-up.svg";
+const chevronDown = "/theme-assets/chevron-down.svg";
+
+for (let i = 0; i < navChevron.length; i++) {
+    let siblingList = navChevron[i].nextElementSibling;
+    navChevron[i].addEventListener('click', () => {
+        if (navChevron[i].children[0].src.indexOf('chevron-down') >= 0) {
+            navChevron[i].children[0].src = chevronUp
+        } else { 
+            navChevron[i].children[0].src = chevronDown
+        }
+        navChevron[i].parentNode.classList.toggle('nav-active');
+        siblingList.classList.toggle('hidden')
+    });
```

Comment:
I feel like a lot of this code is brittle; why not use `navElement` (as discussed earlier) as `navElement.querySelector`? It's already scoped to the parent element and lets you search down.

If we restructure the html a little bit, this could be simplified to toggle on/off an `.active` class that would just change the chevron (and turn on/off hiding), which might be a tad easier on your sanity.